projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -188,7 +190,19 @@ export default function PublicizeForm( {
 							/>
 						</>
 					) }
-					{ isEnhancedPublishingEnabled && <MediaSection /> }
+					{ isEnhancedPublishingEnabled && (
+						<MediaSection
+							disabled={ isImageGeneratorEnabled }
```

Comment:
This is definitely an odd state, but I published a post with SIG enabled and then turned the feature off. That meant that `isImageGeneratorEnabled` here is true, but the panel is hidden as isSocialImageGeneratorEnabled from `usePublicizeConfig` is false.

Do you think we should try and use both of these values? It seems like useImageGeneratorConfig shouldn't come back as enabled if the feature isn't available, although that could mean people lose the OG images if they edit a post after letting the Advanced plan lapse.

<img width="263" alt="image" src="https://user-images.githubusercontent.com/96462/234928520-c71bfe19-b348-413b-81fa-5ce077ac0a37.png">

Reply:
I've added this change in [1996bbd](https://github.com/Automattic/jetpack/pull/30311/commits/1996bbd0840814208c86e4640d14fc9f7ada949b), but it's possibly not a situation that will ever occur!